### PR TITLE
Full Size FPV Drone

### PR DESCRIPTION
**Pull request by ultratrikx**
## Describe your project in 1-3 sentences
I am building a full size 5" FPV drone from scratch. It will have bluetooth connectivity to your phone and be able to record cinematic footage with a camera attached to it. 

## Hack Club Slack handle
rohanth1056

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: ultratrikx.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+I am going to be building a 5" full size FPV (First Person View) drone from scratch. It will be capable for recording cinematic FPV footage after I learn how to fly it. In the process of building this drone I hope to learn and improve on skills such as soldering, electronical engineering, flight design and more. This project will require me to connect all the parts of the drone together and program the flight systems to ensure real world performance. I am excited to build this since I have always wanted to have an FPV drone because of the compelexity and challenge it poses in both flying and building compared to a normal one. Building the drone myself will allow me to learn more about how each component works and how it interacts with the rest of the drone.
+
+## Plan
+
+For most of the building guide I will be following this [video](https://www.youtube.com/watch?v=e0nFgiWlVYk).
+
+The TLDR of the my steps is as follows
+1. Assemble Frame, attach motors and flight controller stack
+2. Solder motor connections with soldering iron
+3. Attacth radio, VTX, camera to frame
```
Comment (bellesea) [inline ultratrikx.md:19]:
```suggestion
3. Attach radio, VTX, camera to frame
```

**Review by bellesea — CHANGES_REQUESTED** [review]
hiii @ultratrikx this is really cool!!

i'm wondering:
- have you confirmed/researched that a go pro would work instead of a Foxeer camera?
- could you please add the products you already own? eg. VTX, goggles, etc.

thank you!

**Comment by ultratrikx** [comment]
> hiii @ultratrikx this is really cool!!
> 
> i'm wondering:
> 
> * have you confirmed/researched that a go pro would work instead of a Foxeer camera?
It is possible to use a GoPro as a feed for the FPV but it results in high latency and thus isn't the best for highspeed maneuvers. While an adequate FPV camera is only $20 (and another $12 for the VTX), it would go beyond the $250 budget given. Regarding this, I was wondering if it would be possible for us to put in some money ourselves to fulfill our build ideas. That way I can use a proper FPV camera instead of a high-latency GoPro.

> * could you please add the products you already own? eg. VTX, goggles, etc.
Will do!

**Review by ultratrikx — COMMENTED** [review]
Updated list of products I own.

**Comment by thinkingjet** [comment]
not from hack club but yo how're u planning on adding the bluetooth support? Like what'll it be used for cus u already have a controller listed in ur products list

**Comment by ultratrikx** [comment]
> not from hack club but yo how're u planning on adding the bluetooth support? Like what'll it be used for cus u already have a controller listed in ur products list

The flight stack has built in bluetooth support which can be used to send flight information such as altitude and speed to a connected device.

**Comment by bellesea** [comment]
hiii @ultratrikx yes you can definitely add some money yourselves :) once you update the budget we should be good to go with moving forward!

**Comment by ultratrikx** [comment]
Awesome, tysm! Will update the budget then.

**Review by bellesea — APPROVED** [review]
@ultratrikx ahh love this!! 

an awesome member of @hackclub/bank-ops will be reaching out soon with next steps!

ps. approving this to move forward but for the items you already have could you pease make it $0 and it's ok for the total to be over $250, we'd just give you up to $250 :) thank you sm!

**Review by daisybanks — APPROVED** [review]
@ultratrikx wow this is going to be one big drone! I'm dming you now on Slack to talk about next steps in getting you you're grant! :)

**Pull request merged**